Adyen/Integration/01_ManageAdyenSettings.md

```diff
@@ -0,0 +1,282 @@
+
+       
+    ![Create API credential](../../Assets/Screenshots/Adyen/Integration/AdyenCreateAPIcredential.png "[Create API credential]")
+   
+  2. Select the *Report service user* credential type.   
+   The username is automatically created by the system.
+
+       ![Create report service credentials](../../Assets/Screenshots/Adyen/Integration/AdyenReportServiceAPIcredential.png "[Create report service credentials]")
+
+3. Copy the username.
+
+4. Change to your Actindo instance and paste the report username in the *Report user* field.
+  
+   ![Report username](../../Assets/Screenshots/Adyen/Integration/AdyenReportUser.png "[Report username]")
+
+4. Return to your Adyen&trade; account. 
+
+5. Enter a description for the report service user in the *Description* field.
+
+    ![Create report service credentials](../../Assets/Screenshots/Adyen/Integration/AdyenReportServiceAPIcredential.png "[Create report service credentials]")
+
+6. Click the [Create credential] button.   
+    The *Configure API credential* view is displayed. The API key tab is displayed by default.
+
+7. Generate the *Report service* API key and password. To do this, follow these steps:
+
+    - Click the *API key* tab in the *Server settings > Authentication* section.
+    
+    - Click the [Generate API key] button.   
+        The API key is automatically applied.  
+        
+        ![Configure API credentials](../../Assets/Screenshots/Adyen/Integration/AdyenReportAPICredentials.png "[Configure API credentials]")
+
+        > [Info] You do not need to enter the report API key in your Payments connection settings. For the *Payments* module, the report name and password are sufficient.
+
+    - Click the *Basic auth* tab in the *Server settings > Authentication* section.
+   
+    - Click the [Generate password] button.   
+      The password is automatically created.
+   
+       ![Report service password](../../Assets/Screenshots/Adyen/Integration/AdyenWebServicePassword.png "[Report service password]")
+
+    - Click the [Copy] button.
+
+7. Change to your Actindo instance and insert the password in the *Report password* field. 
+
+   ![Report service password](../../Assets/Screenshots/Adyen/Integration/AdyenReportServicePassword.png "[Report service password]")
+
+8. Return to your Adyen&trade; account and click the [Save changes] button in the bottom right corner.   
+    The credentials for the *Report service* API are applied.
+
+9. If necessary, assign the required report role to allow the technical user to download reports in the *Permissions > Roles > REPORT* section. Additionally, you can control access to merchant accounts. For detailed information, see [API permissions](https://docs.adyen.com/development-resources/api-credentials/#api-permissions "[https://docs.adyen.com/development-resources/api-credentials/#api-permissions]") in the Adyen&trade; documentation.
```

Comment:
Was ist ein technical user? Admin user? Gibt es die Unterscheidung in Adyen? Macht es Sinn, bei uns zu unterscheiden? (Bei uns gibt es noch nicht, sofern ich weiß). Ich hinterfrage, weil wir das bisher nicht haben. Wenn wir es verwenden, sollten wir es erklären, finde ich.